tool/icons/update_icons.dart

```diff
@@ -5,20 +5,22 @@
 import 'dart:convert';
 import 'dart:io';
 
-const outputFolder = 'resources/icons';
+import '../common.dart';
 
-void main() async {
+const String outputFolder = 'resources/icons';
+
+Future<void> main() async {
   // download material/icons.dart and cupertino/icons.dart
-  String materialData =
+  final String materialData =
```

Comment:
Can you add a TODO: here to instead copy from the local flutter sdk?

Reply:
Oh yeah - there's a few places we're downloading that we should now pull from the local one. I've added TODOs to each of them. Thanks!